internal/service/project/resource_project_test.go

```diff
@@ -980,12 +980,87 @@ func TestAccProject_withInvalidLimitNameOnUpdate(t *testing.T) {
 	})
 }
 
+func TestAccProject_withTags(t *testing.T) {
+	var (
+		orgID        = os.Getenv("MONGODB_ATLAS_ORG_ID")
+		projectName  = acc.RandomProjectName()
+		nameUpdated  = "my-tag-name-updated"
+		envUnchanged = "unchanged"
+		tags1        = map[string]string{
+			"Name":        "my-tag-name",
+			"Environment": envUnchanged,
+			"Deleted":     "short-lived",
+		}
+		tagsOneUpdatedOneDeleted = map[string]string{
+			"Name":        nameUpdated,
+			"Environment": envUnchanged,
+			"NewKey":      "new-value",
```

Comment:
referring of what I was saying before, one last step could be to check if a >255char long tag fails

Reply:
Added this and got a 500 :S
https://jira.mongodb.org/browse/CLOUDP-242986